docs(readme): add asset information description (#49)
LemoFoundationLtd/lemo-client: README.md

lemo.TxType.CREATE_ASSET | 3 | Create asset information
lemo.TxType.ISSUE_ASSET | 4 | Issue asset
lemo.TxType.REPLENISH_ASSET | 5 | Replenish asset transaction
lemo.TxType.MODIFY_ASSET | 6 | Modify asset transaction
lemo.TxType.TRANSFER_ASSET | 7 | Trading assets

chainID | description
---|---
1 | LemoChain main net
100 | LemoChain test net

<a name="data-structure-asset"></a>
#### asset
Asset information
```json
{
    "category": "1",
    "decimals": 18,
    "totalSupply": "15000000000000000000",
    "isReplenishable": true,
    "isDivisible": true,
    "issuer": "Lemo83GWNWJQ3DQFN6F24WFZF5TGQ39A696GJ7Z3",
    "profile": {
        "name": "Demo Asset",
        "symbol": "DT",
        "description": "this is a asset information",
        "suggestedGasLimit": "60000",
        "stop": "false"
    }
}
```
- `category` Asset type
- `decimals` The decimal place of the issued asset, which indicates how many digits are reduced to the decimal point. The default is 18 digits.
- `totalSupply` The total amount of assets issued, when the assets are issued and destroyed, the total amount of assets will change in real time, the total amount of assets issued is `issuance*10^decimals`

> - `isReplenishable` Whether the asset can be replenish, The asset cannot be modified again after set， set when the asset is created, the default is `true`

Reviewer: ```suggestion
- `isReplenishable` Whether the asset can be replenish. It is not changeable after setup when creating the asset. The default is `true`
```

- `isDivisible` Whether the asset is divisible, set when the asset is created, the default is `true`

> - `issuer` The publisher address, based on the signature field at the time the asset was created

Reviewer: ```suggestion
- `issuer` The publisher address. It is not settable
```

- `profile` Additional information about the asset

>     - `name` The name of the asset when it was created

Reviewer: ```suggestion
    - `name` The name of the asset
```

    - `symbol` Asset identification
    - `description` Basic information of assets

>     - `suggesteGasLimit` Trading consumption limit, and ` gasLimit ` same usage, creating assets set by users themselves, the default is 60000

Reviewer: ```suggestion
    - `suggestedGasLimit` Suggested gas limit. It is like the `gasLimit` field in transaction. The default is 60000
```

>     - `stop` Whether to stop the special deals, the default is ` false `, if there is a loophole, set it to ` true ` can stop any special deal

Reviewer: ```suggestion
    - `stop` Whether to stop all actions of the asset. The default is ` false`
```

}
```
- `category` Asset type
- `decimal` The decimal place of the issued asset, which indicates how many digits are reduced to the decimal point. The default is 18 digits.
- `totalSupply` The total amount of assets issued, when the assets are issued and destroyed, the total amount of assets will change in real time, the total amount of assets issued is `issuance*10^decimal`
- `isReplenishable` Whether the asset can be replenish. It is not changeable after setup when creating the asset. The default is `true`
- `isDivisible` Whether the asset is divisible, set when the asset is created, the default is `true`
- `issuer` The publisher address. It is not settable
- `profile` Additional information about the asset
    - `name` The name of the asset
    - `symbol` Asset identification
    - `description` Basic information of assets
    - `suggesteGasLimit` Suggested gas limit. It is like the `gasLimit` field in transaction. The default is 60000
    - `stop` Whether to stop all actions of the asset. The default is ` false`

<a name="data-asset-category"></a>

category | description
--- |---

> 1 |  TokenAsset, similar to ethereum's ERC20 token, it can set whether additional issuance is allowed in the future when issuing, and users can divide and transfer money at will after receiving the coin

Reviewer: 1 |  TokenAsset. Similar to Ethereum's ERC20 token. The issuer can decide whether additional issuance is allowed in the future when issuing. The token is divisible.
2 |  NonFungibleAsset. Similar to Ethereum's ERC721 token, it can store a certain amount of information. Every token is indivisible
3 |  CommonAsset. More flexible digital assets. It is suitable for more complex scenarios. Actualy, the TokenAsset and NonFungibleAsset are two special CommonAsset

    - `symbol` Asset identification
    - `description` Basic information of assets
    - `suggesteGasLimit` Suggested gas limit. It is like the `gasLimit` field in transaction. The default is 60000

>     - `stop` Whether to stop all actions of the asset. The default is `false`. Set it to `true` can stop all the assets other than the query operation

Reviewer: - `stop` Whether to freeze asset. The default is `false`. Set it to `true` will stop all actions about the asset except query operations

<a name="data-asset-category"></a>

category | description
--- |---
1 |  TokenAsset. Similar to Ethereum's ERC20 token. The issuer can decide whether additional issuance is allowed in the future when issuing. The token is divisible.
2 |  NonFungibleAsset. Similar to Ethereum's ERC721 token, it can store a certain amount of information. Every token is indivisible.
3 |  CommonAsset. More flexible digital assets. It is suitable for more complex scenarios. Actualy, the TokenAsset and NonFungibleAsset are two special CommonAsset.

<a name="data-structure-equity"></a>

#### equity

Record the asset information of the asset holder and keep it in the account of the asset holder

```json
{
    "assetCode": "0xd0befd3850c574b7f6ad6f7943fe19b212affb90162978adc2193a035ced8884",
    "assetId": "0x34b04e018488f37f449193af2f24feb3b034c994cde95d30e3181403ac76528a",
    "balance": "15000000000000000000"
}
```

-   `assetCode` Asset code, obtained when the asset is created

> -   `assetId` The assetId, which is obtained when the asset is issued, is the same assetCode as the assetId when the asset type is 1

Reviewer: -   `assetId` The assetId, which is obtained when the asset is issued. It will be same as assetCode when the asset type is 1
